README.md

```
@@ -1,3 +1,127 @@
+- [x] 기물의 도착 위치 입력 `기물이 도착할 위치를 입력해주세요. (ex. 1,2)`
+- [x] 입력 위치에 따른 기물 이동
+- [x] 장기판 상태 출력
+- [x] `예외처리` 기물이 이동할 수 없는 위치인 경우 예외가 발생한다 `[ERROR] 기물이 이동할 수 없는 위치입니다.`
+- [x] `예외처리` 잘못된 좌표 입력인 경우 예외가 발생한다 `[ERROR] 좌표 입력은 9X10 보드 이내만 가능합니다.`
+
+### ✅ 게임 승리 기능
+
+- [x] 상대의 궁을 잡은 경우
+- [x] 게임 종료 후 우승 메시지 출력
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)